src/import/client.rs

```diff
@@ -290,7 +290,8 @@ impl<R: Read> Stream for UploadStream<R> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use rand::{self, Rng};
+    use rand;
```

Comment:
`use rand` is not necessary in Rust 2018

Reply:
Thx! I will fix it.